content/forms/openpower-gpu-request.rst

```diff
@@ -0,0 +1,104 @@
+              <option value="Fedora">Fedora</option>
+              <option value="CentOS">CentOS</option>
+              <option value="Debian">Debian</option>
+              <option value="Ubuntu">Ubuntu</option>
+              <option value="OpenSUSE">OpenSUSE</option>
+              <option value="Other">Other</option>
+            </select>
+            <div class="description">Which Linux distribution would you like to use for your machine? This would likely be the latest stable version available for PPC. If you want a specific version, please state that in the comments section on the last page.</div>
+          </div>
+          <div class="form-item webform-component webform-component-textarea" id="webform-component-ssh-key">
+            <label for="edit-submitted-ssh-key">SSH Public Key <span class="form-required" title="This field is required.">*</span></label>
+            <div class="form-textarea-wrapper resizable"><textarea id="edit-submitted-ssh-key" name="ssh_public_key" cols="60" rows="5" class="form-textarea required"></textarea></div>
+            <div class="description">Public SSH key to be used for initial access to the system.</div>
+          </div>
+          <div class="form-item webform-component webform-component-select" id="webform-component-deployment-timeframe">
+            <label for="edit-submitted-deployment-timeframe">Deployment timeframe </label>
+            <select id="edit-submitted-deployment-timeframe" name="deployment_timeframe" class="form-select">
+              <option value="Within 7 business Days" selected="selected">Within 7 business Days</option>
+              <option value="Within 3 business Days">Within 3 business Days</option>
+              <option value="Within 1 business Days">Within 1 business Day</option>
+            </select>
+            <div class="description">Normal turnaround for access is typically 7 business days. If you need it sooner than that, please choose which time frame you need. We will do our best to accommodate your request. </div>
+          </div>
+          <div class="form-item webform-component webform-component-textarea" id="webform-component-other-information">
+            <label for="edit-submitted-other-information">Other information </label>
+            <div class="form-textarea-wrapper resizable"><textarea id="edit-submitted-other-information" name="other_information" cols="60" rows="5" class="form-textarea"></textarea></div>
+            <div class="description">Is there anything additional you would like to provide for your request?</div>
+          </div>
+
+          <p><i>You should receive an automated email from our request ticketing system to the email address you have provided
+          within 5-10 minutes.  If you don't receive this email please reach out to us at <a href="mailto:openpower-gpu-support@osuosl.org">openpower-gpu-support@osuosl.org</a> or
+          via IRC in <b>#osuosl</b> on Freenode.</i></p>
```

Comment:
We should ask Chris or Ian about a different IRC channel or other form of communication.